aten/src/ATen/cpu/vec256/vec256_bfloat16.h

```diff
@@ -2,7 +2,7 @@
 
 #include <ATen/cpu/vec256/intrinsics.h>
 #include <ATen/cpu/vec256/vec256_base.h>
-#if defined(__AVX__) && !defined(_MSC_VER)
+#if defined(CPU_CAPABILITY_AVX2) && !defined(_MSC_VER)
```

Comment:
This should be AVX | AVX2, isn't it?
```suggestion
#if (defined(CPU_CAPABILITY_AVX) || defined(CPU_CAPABILITY_AVX2)) && !defined(_MSC_VER)
```

Reply:
So this was also a little fix I was sneaking in. If you look further down, the whole file contents are `ifdef` on `AVX2`, so the extra include is also only needed when `AVX2` is defined. But maybe this was intentional? maybe other headers rely on this to get sleef even though this file would not use it at all in the `AVX` case.